.travis.yml

```diff
@@ -12,6 +12,7 @@ python:
   - "2.7"
 before_script:
   - sleep 10
+  - echo -e "https://github.com/cyberreboot/vent:\n  rabbitmq:\n    build: yes\n    start: yes" >> ~/.vent_startup.yml
```

Comment:
travis doesn't like this for some reason.  looks like want it like this with `-e`: https://docs.travis-ci.com/user/customizing-the-build#Authentication

Reply:
I'm confused, I saw that guide and modeled it after that, but I'm still getting issues.

Reply:
oh, you're right.  that's really strange.